echopype/echodata/convention/echodata_structures.py

```diff
@@ -0,0 +1,327 @@
+#############################################################################################
+# PLEASE CAREFULLY REVIEW THE RULES BEFORE APPLYING CHANGES.
+#
+# Rules of the structure:
+#
+# 1. For each sensor, the keys of each group should correspond to the groups in 1.0.yml
+# 2. The ep_group should correspond to the group that would be used to access the DataTree.
+# 2. coords should contain all POSSIBLE unique coordinates.
+# 3. If you add a version structure, please add this version to all sensors.
+# 4. For each coords value carefully compare and modify all corresponding
+# coords of the other groups. For example, if you were to add 'beam' to coords, giving you
+# "coords": ['channel', 'ping_time', 'range_sample', 'beam'], then you should add
+# None to any coords that do not have that coordinate (in that sensor). For exampple,
+# "coords": ['frequency', 'ping_time', 'range_bin', None]
+# 5. If the group does not apply to the sensor, you should replace the dict with None
+#############################################################################################
+
+SONAR_STRUCTURES = {
```

Comment:
- Add a description of what this dictionary is and what it's for
- SONAR_STRUCTURES seems fairly generic. Though, anything truly descriptive would probably get pretty long! A possibility: CONVENTION_IMPLEMENTATION_MAPPINGS (yes, pretty long)

Reply:
> Add a description of what this dictionary ...

Sounds good, I will be sure to add this. 

> SONAR_STRUCTURES seems fairly generic.

Yes, this is pretty generic. How about `SENSOR_VERSION_MAPPINGS`?

Reply:
It's getting there. How about this slightly longer version: `SENSOR_EP_VERSION_MAPPINGS` (EP = echopype)

Reply:
It is a little long, but I think `SENSOR_EP_VERSION_MAPPINGS` works for me. I will go ahead and change it to this.